sigle/src/modules/layout/components/DashboardLayout.tsx

```diff
@@ -27,12 +27,14 @@ import { toast } from 'react-toastify';
 import { useAuth } from '../../auth/AuthContext';
 
 export const DashboardContainer = styled(Container, {
+  px: 0,
```

Comment:
@gregogun issue is that by removing the padding of the main container, you are impacting all the screens depending on the dashboard layout

![Screenshot 2022-10-19 at 13 14 29](https://user-images.githubusercontent.com/5749437/196676102-21cf19c5-25ba-41a1-bf66-d7a084d5b780.png)